products/vmware_cb_response.py

```diff
@@ -59,14 +68,18 @@ def process_search(self, tag: Tag, base_query: dict, query: str) -> None:
             for proc in self._conn.select(Process).where(query):
                 result = Result(proc.hostname.lower(), proc.username.lower(), proc.path, proc.cmdline,
                                 (proc.start, proc.id))
+                raw_results.append(proc)
                 results.add(result)
```

Comment:
To avoid duplication

```suggestion
                if self._raw:
                    results.add(proc)
                else:
                    results.add(result)
```